source/clear-linux/tutorials/kata.rst

```diff
@@ -28,24 +28,23 @@ Install Kata Containers
 ***********************
 
 Kata Containers is included in the :file:`containers-virt` bundle. To install the
-framework, enter:
+framework, enter the following command:
 
 .. code-block:: bash
 
    sudo swupd bundle-add containers-virt
 
-Configure Docker\* to use Kata Containers by default
+Configure Docker\* to use Kata Containers by default.
 
 .. code-block:: bash
 
    sudo mkdir -p /etc/systemd/system/docker.service.d/
-   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/51-runtime.conf
    [Service]
```

Comment:
@joselamego, @devimc 
Can you please confirm that the cat command on Line 42 is correct? It ends in "...51-runtime.conf"

Reply:
LGTM

Reply:
Thank you.

Reply:
@mythi.  Thank you for testing and confirming.